### ISSUE TITLE
EC2 - gmond on node cannot communicate with gmetad on master

### ISSUE THREAD

**Issue by yaron-idan** [ganglia/monitor-core#238]
I am trying to use Ganglia to monitor our spark activity.
gmetad successfully collects data from the gmond running on the same server, but when data from the slave is not available.
Telnet and nc succeeds between master and node, and gmond runs properly on the node server.

One peculiar detail - When examining the XML coming from the node server, the ip is the same as the master server, which makes me think something is wrong with my configuration.
If any configuration files or part of them are required let me know and I'll upload them.

Any help would be greatly appreciated.
Thank you,
Yaron.

**Comment by vvuksan** [comment]
Were you able to resolve this issue ?

**Comment by yaron-idan** [comment]
Yes, the problem was not commenting the mcast_join when using unicast.
Thanks!
